app/models/notification.rb

```diff
@@ -0,0 +1,5 @@
+#  Modelo para las notificaciones del sistema
+class Notification < ApplicationRecord::Base
+  belongs_to :follower, class_name: 'user'
```

Comment:
A notification should only be associated to the user that generates the notification, otherwise if an user does an action, should a notification be created for each follower? It's user to just create one and then keep tabs on who's read the notification through the ReadNotification table